CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 5.1.0 (unreleased version)
+#### Notes
+Provides SDK support to OneView REST API version 800, 1000 and 1200 (OneView v5.10).
```

Comment:
we already had a release to support API versions 800, 1000 and 1200. The description should be updated. Are we supporting OneView version 5.10?

Reply:
Updated the oneview version and changed the description